R/plot.R

```diff
@@ -61,14 +61,18 @@ plot.biokinetics_priors <- function(x,
 
 plot_sero_data <- function(data, covariates = character(0)) {
   validate_required_cols(data, c("time_since_last_exp", "value", "titre_type"))
+  add_censored_indicator(data)
   # Declare variables to suppress notes when compiling package
   # https://github.com/Rdatatable/data.table/issues/850#issuecomment-259466153
-  time_since_last_exp <- value <- titre_type <- NULL
+  time_since_last_exp <- value <- titre_type <- censored <- NULL
 
   ggplot(data) +
-    geom_point(aes(x = time_since_last_exp, y = value, colour = titre_type)) +
-    geom_smooth(aes(x = time_since_last_exp, y = value, colour = titre_type)) +
+    geom_point(aes(x = time_since_last_exp, y = value, colour = titre_type, shape = censored),
+               size = 0.5, alpha = 0.5) +
+    geom_smooth(data = subset(data, !censored),
```

Comment:
Added the censored indicator in to the input data plot as well, and excluding censored values from the spline - do we think this is the best way to deal with these data points? In the legacy data, the censored values are for some reason much lower than any other value in the dataset (so presumably lower than the actual assay detection limit), but if this wasn't the case would probably make sense to include them,.

Reply:
yeah the shape is different if censored, but dashed lines could be good too :+1: